src/sparseml/pytorch/sparsification/pruning/modifier_pruning_base.py

```diff
@@ -598,9 +598,6 @@ def __init__(
             log_types=log_types,
             global_sparsity=global_sparsity,
             allow_reintroduction=allow_reintroduction,
-            init_sparsity=self._init_sparsity_orig,
-            final_sparsity=self._final_sparsity,
-            inter_func=inter_func,
```

Comment:
these should be left in to be propagated as kwargs through `BasePruningModifier`, necessary for multiple inheritance with other classes

Reply:
Should these then be added to `BasePruningModifier`? Because right now they're not caught by any subclass, so they're propagated through to BaseObject, which throws an error.